applications/web/values.yaml

```diff
@@ -19,7 +19,7 @@ terminationGracePeriodSeconds: 30
 
 image:
   repository: public.ecr.aws/o1j4x7p4/hello-porter
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
instead of hardcoding it can we set it to  `{{ .Values.image.pullPolicy }}` and then set `Always` as the default?